helm-chart/binderhub/templates/ingress.yaml

```diff
@@ -5,30 +5,39 @@ metadata:
   name: binderhub
   {{- if or (and .Values.ingress.https.enabled (eq .Values.ingress.https.type "kube-lego")) .Values.ingress.annotations }}
   annotations:
-    {{- if eq .Values.ingress.https.type  "kube-lego" }}
+    {{- if and .Values.ingress.https.enabled (eq .Values.ingress.https.type "kube-lego") }}
```

Comment:
To avoid template rendering failures i added default values for `ingress.https.enabled: false` and `ingress.https.type: kube-lego`.